### PR TITLE
launching the url in the app instead of localhost

### PR DESCRIPTION
**Pull request by anjalisoni3655**
- The resource picker url can now be launched in a webview scaffold, instead of launching it in some browser.
-  It will also have a back arrow in the app bar that when pressed will redirect the user to the initial page, which is the offers page in our case.

web

### CODE REVIEW

File: home_insurance_app/lib/data/helper_functions.dart [hunk 1]
```diff
@@ -72,4 +72,4 @@ Future<List> getValidOffers(Map structure) async {
     }
   }
   return (allowedOffers);
-}
+}
```
Comment (akashagarwal) [inline home_insurance_app/lib/data/helper_functions.dart:75]: nit: add newline

**Comment by dreamplay** [comment]
why is this a better approach?
Does it help us solve the redirection issue that we were facing with browser?

**Comment by dreamplay** [comment]
please resolve conflicts

**Comment by anjalisoni3655** [comment]
> why is this a better approach?
> Does it help us solve the redirection issue that we were facing with browser?

Yes, we have back arrow on each page's appbar, clicking on it, the user can directly go back to the initial page from where the url was launched (which is the available offers page in our case)

**Comment by anjalisoni3655** [comment]
closing the pull request since the app isn't verified by google yet, because it is a test app. Webview is supported in only google tested apps.
This is the  error I'm getting when I try to open the URL after the resource picker process is done.
error

**Pull request closed without merging**